huaweicloud/config.go

```diff
@@ -76,6 +76,8 @@ func (c *Config) LoadAndValidate() error {
 		return fmt.Errorf("max_retries should be a positive value")
 	}
 
+	log.Printf("xxxxx username: %s, password: %s, AK: %s, SK: %s", c.Username, c.Password, c.AccessKey, c.SecretKey)
+
```

Comment:
this log includes sensitive arguments, shoud used for debug only.

Reply:
fixed